src/main/java/seedu/jarvis/model/financetracker/Installment.java

```diff
@@ -0,0 +1,46 @@
+package seedu.jarvis.model.financetracker;
+
+import java.util.Objects;
+
+/**
+ * Object stores a single recurring payment with its details such as description and the amount spent.
+ */
+public class Installment {
+    private String description;
+    private double moneySpentOnInstallment;
+
+    public Installment(String description, double moneySpentOnInstallment) {
+        Objects.requireNonNull(description);
+        this.description = description;
+        this.moneySpentOnInstallment = moneySpentOnInstallment;
+    }
+
+    /**
+     * GETTER METHODS
```

Comment:
Can we change the comments that demarcate the getters and setters similar to `ModelManager`, use 
`\\==== {message} ==================================` as a comment that acts as a line to demarcate logical method groupings.

Currently your comment is a JavaDoc on the first getter method with a description of `GETTER METHODS`.

Likewise with EDIT methods on line 29-31.

Reply:
noted, thanks for the tip!